voidcrew/code/game/machinery/computer/shuttle/auto_pilot.dm

```diff
@@ -74,6 +100,45 @@
 	if(!length(data["locations"]))
 		data["locked"] = TRUE
 		data["status"] = "Locked"
+	/*
+		Subverter ui data
+	*/
+
+	data["subverter"] = sub
+	if (sub)
+		data["ships"] = list()
+		data["locked_b"] = FALSE
+		data["recharge_time_str"] = (sub.subverter_cooldown - world.time) > 0 ? sub.get_recharge_str() : "--:--"
+		data["selected_target"] = subvert_target
+		data["status_b"] = "Ready"
+		if(!ship?.shuttle)
+			data["status_b"] = "Missing"
+			return data
+
+		switch(ship.state)
+			if(OVERMAP_SHIP_UNDOCKING)
+				data["locked_b"] = TRUE
+			if(OVERMAP_SHIP_DOCKING)
+				data["locked_b"] = TRUE
+
+		for(var/obj/structure/overmap/ship/simulated/O in view(ship.sensor_range, get_turf(ship)))
+			if(O == ship)
+				continue
+			var/list/location_data = list(
+				id = REF(O),
```

Comment:
refs on non directly related things is a good way to lead to harddels, maybe use a weakref here?

Reply:
thats just how the autopilot console was but ill look into it

Reply:
hmm okay it's probably fine then